### DISCUSSION TITLE
Copying code from documentation could be easier in some places

### DISCUSSION THREAD

**Discussion by gouravsaini021** [encode/httpx#2128] [Ideas]
when we copy code from the documentation it also copies `>>>`  and `...`, which makes the experience bad.   

One solution could be to use comments. So instead of 
```
>>> async with httpx.AsyncClient() as client:
...     r = await client.get('https://www.example.com/')
...
>>> r
<Response [200 OK]>
```
we can have
```py
async with httpx.AsyncClient() as client:
     r = await client.get('https://www.example.com/')
print(r)  # <Response [200 OK]>
```
```

**Comment by johtso** [comment]
I can't remember exactly where I saw it, but I've seen some documentation have a little button in the top right corner to toggle the "command line characters" on or off.
